livekit/types.go

```diff
@@ -50,3 +50,12 @@ type Guid interface {
 }
 
 type GuidBlock [9]byte
+
+// ToProto implements DataPacket in Go SDK.
+func (p *SipDTMF) ToProto() *DataPacket {
```

Comment:
See the SDK PR for details: https://github.com/livekit/server-sdk-go/pull/415/files#diff-367620201829307624e013978caae5ed6ce4f0b96630e075e8760e0a04a41746R306